src/app/(core)/v/[slug]/player.tsx

```diff
@@ -86,18 +86,33 @@ export const VodPlayer = (props: { id: string; vod: VOD }) => {
     ...props.vod.markers,
   ];
 
-  const [offset, setOffset] = useState(0);
-
+  const [offset, setOffset] = useState<{presentational: string, totalSeconds: number}>({
```

Comment:
this breaks output csv file names, need to change L135 to reflect this

Reply:
good catch, should be fixed now.